NEWS

```diff
@@ -1,7 +1,48 @@
+KLEE 3.0, xx May 2023
+======================
+
+Incorporating changes from 8 April 2022 to xx May 2023.
+Maintainers during this time span: @ccadar, @MartinNowack, @251.
+(@251 joined as a maintainer in 2023).
+Documentation at http://klee.github.io/releases/docs/v3.0
+
+== Major features ==
+- Added support for the KDAlloc memory allocator, which enables KLEE to more robustly detect use-after-free errors, improves the detection of buffer overflows, and provides deterministic memory allocation (@@danielschemmel, based on https://srg.doc.ic.ac.uk/publications/22-kdalloc-ecoop.html)
```

Comment:
`@@`

Reply:
Good catch, although he might deserve a double @ for integrating KDAlloc! ;)